docs/integrations/development.md

```diff
@@ -297,7 +297,18 @@ await iterateEntities({ _type: 'my_integration_user' }, async (userEntity) => {
 ```
 
 Specific entities can be looked up using the entity `_key` property via the
-`getEntity` function.
+`findEntity` function. `findEntity` will return `null` if the entity does not
+exist.
+
+Example usage:
+
+```typescript
+const entity = await jobState.findEntity('myentitykey');
+```
+
+Similarly to `findEntity`, `getEntity` can also be used to look up an entity by
+`_key`. `getEntity` will throw a `IntegrationDuplicateKeyError` error if the
+entity already exists.
```

Comment:
I think this should be `IntegrationMissingKeyError`.

Related: looks like the [test job state does not throw the right kind of error](https://github.com/JupiterOne/sdk/blob/c93f69eb0fd90012e58f5accfe1f7c1b8bdf7634/packages/integration-sdk-testing/src/jobState.ts#L122).

I'm curious whether there is a way we can avoid a "mock" job state class at some point, because I've seen a couple times where it doesn't do the same thing as the real one.